pkgs/tools/misc/spigot/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, buildPackages
+, fetchurl
+, gmp
+}:
+
+stdenv.mkDerivation rec {
+  name = "spigot-20200101";
+  src = fetchurl {
+    url = "https://www.chiark.greenend.org.uk/~sgtatham/spigot/${name}.b1b0b20.tar.gz";
+    sha256 = "0cf9h6x2n016ngy5qx53kkmgi4v8a3zinmh534j4gafmmzap061s";
+  };
+
+  configureFlags = [ "--with-gmp" ];
+
+  depsBuildBuild = [ buildPackages.stdenv.cc ];
+  nativeBuildInputs = [
+    # Tools
+    buildPackages.gmp
+  ];
+  buildInputs = [ gmp ];
+
+  doCheck = true; # not cross
+
+  strictDeps = true;
+
+  meta = with lib; {
+    description = "a command-line exact real calculator";
```

```suggestion
    description = "A command-line exact real calculator";
```

https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes